styles/style.css

```diff
@@ -0,0 +1,226 @@
+    margin-right: 5%;
+    column-gap: 30px;
+}
+
+.navbar a {
+    text-decoration: none;
+    display: block;
+    padding: 1.5em;
+    color: #4CA1A3;
+}
+
+.navbar a:hover {
+    color: white;
+}
+
+/* Medium screens */
+@media all and (max-width: 800px) {
+    .navbar {
+        justify-content: space-around;
+    }
+}
+
+/* Small screens */
+@media all and (max-width: 500px) {
+    .navbar {
+        flex-direction: column;
+    }
+}
+
+/* Main-section link styles */
+#main-section {
+    height: 100%;
+    margin: 13%;
+    text-align: left;
+}
+
+.button {
+    background-color: transparent;
+    border: 2px solid white;
+    color: white;
+    opacity: 0.6;
+    padding: 10px 20px;
+    border-radius: 12px;
+    font-size: 0.6em;
+    text-decoration: none;
+}
+
+.button:hover {
+    background-color: #6F39B9;
+    opacity: 0.8;
+}
+
+/* Styles for footer header and paragraph */
+footer {
+    text-align: center;
+}
+
+.footer-section h2 {
+    font-size: 0.7em;
+    color: white;
+    opacity: 0.6;
+}
+
+.footer-section p {
+    font-size: 0.5em;
+}
+
+/* Styles for footer social links and copyright line */
+
+.social-media-link {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    list-style: none;
+    margin-bottom: 0;
+}
+
+.social-media-link a {
+    color: #4CA1A3;
+    font-size: 0.6em;
+    padding: 10px;
+    text-decoration: none;
+}
+
+.social-media-link a:hover {
+    color: white;
+}
+
+.social-media-link i,
+.fa-copyright {
+    font-size: 0.9em;
+    padding: 5px;
+}
+
+/* Styles for about page - main section (grid) */
+.grid {
+    background-color: #23033b;
+    display: grid;
+    width: 100%;
+    height: 40%;
+    grid-template: auto 1fr 1fr auto / 1fr 1fr 1fr 1fr 1fr 1fr;
+}
+
+.grid h1 {
+    margin: 0;
+    padding-top: 3%;
+    grid-column: 1/7;
+    grid-row: 1/2;
+    padding-left: 1em;
+    text-align: left;
+}
+
+#grid-img {
+    grid-column: 1/2;
+    grid-row: 2/4;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-left: 0.8em;
+}
+
+.grid img {
+    width: 250px;
+    height: 250px;
+}
+
+.grid-text {
+    grid-column: 2/7;
+    grid-row: 2/4;
+    font-size: 0.6em;
+    padding: 5%;
+}
+
+.grid-skills {
+    grid-column: 1/3;
+    grid-row: 4/5;
+    padding-left: 15%;
+}
+
+.grid-toolset {
+    grid-column: 3/7;
+    grid-row: 4/5;
+    padding-left: 5%;
+}
+
+.grid-skills,
+.grid-toolset {
+    font-size: 0.6em;
+    text-align: left;
+    color: white;
+    opacity: 0.6;
+    padding-bottom: 1em;
+}
+
+/* Styles for portfolio page */
+.projects {
+    text-align: center;
+    margin-bottom: 10%;
+}
+
+.projects>h1 {
+    margin-left: 80px;
+    margin-bottom: 80px;
+    text-align: left;
+}
+
+.projects-flex {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+    font-size: 0.5em;
+    row-gap: 50px;
+}
+
+.projects-flex a:hover {
+    background-color: #511281;
+}
```

Comment:
Neat effect!  Small note, using the selector ".projects-flex a:hover" here and the selector ".projects a" below is a bit confusing when these styles apply to the same "a" tags.  I recommend using a matching selector to improve readability.